hack/vendor.sh

```diff
@@ -55,10 +55,27 @@ clone git github.com/docker/libtrust 230dfd18c232
 
 clone git github.com/Sirupsen/logrus v0.7.2
 
-clone git github.com/go-fsnotify/fsnotify v1.2.0
+clone git github.com/go-fsnotify/fsnotify v1.0.4
 
 clone git github.com/go-check/check 64131543e7896d5bcc6bd5a76287eb75ea96c673
 
+clone git github.com/philhofer/fwd 899e4efba8eaa1fea74175308f3fae18ff3319fa
+
+clone git github.com/tinylib/msgp 75ee40d2601edf122ef667e2a07d600d4c44490c
```

Comment:
Generated code `fluent/proto_gen.go` imports `github.com/tinylib/msgp` to use many interfaces and methods.